config.php

```diff
@@ -4,8 +4,11 @@
 require_once('constants.php');
 
 
-if (!defined('LDAP_HOST'))
-    define('LDAP_HOST', 'pm-ns.mozilla.org');
+if (!defined('LDAP_HOST')) {
+    header('HTTP/1.0 500 LDAP Error');
```

Comment:
HTTP/1.0 and HTTP/1.1 section 6.1.1 both define the text section of "500 xxx" as arbitrary* text designed solely for human use, so this provides us immediate triage debugging in case of an outage and comes at no harm to users.

```
       Reason-Phrase  = *<TEXT, excluding CR, LF>
```